app/dcs/third-party-tools/rclone/rclone-s3/page.md

```diff
@@ -162,8 +162,29 @@ Or between two Storj buckets.
 rclone sync --progress waterbear-us:mybucket/videos/ waterbear-europe:mybucket/videos/
 ```
 
-Or even between another cloud storage and Storj.
+Or even between another cloud storage (e.g., an AWS S3 connection names `s3`) and Storj.
 
 ```bash
 rclone sync --progress s3:mybucket/videos/ waterbear:mybucket/videos/
 ```
+
+
+## Mounting a Bucket
+
+Use the `mount` command to mount a bucket to a folder (Mac and Linux only). When mounted, you can use the bucket as a local folder.
+
+```bash
+sudo mkdir /mnt/mybucket
+sudo chown $USER: /mnt/mybucket
+rclone mount waterbear:mybucket /mnt/mybucket --vfs-cache-mode full
+```
+
+{% callout type="info"  %}
+The `--vfs-cache-mode full` flag means that all reads and writes are cached to disk. Without it, reads and writes are done directly to the Storj bucket.
+{% /callout %}
+
+To unmount the bucket, use the `umount` command.
+
+```bash
+umount /mnt/mybucket
+```
```

Comment:
~~~suggestion
To unmount the bucket, use the `Ctrl-C` keystroke to stop rclone.
~~~